sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -455,9 +455,15 @@ public partial class AzureApplicationInsightsDataFeedSource : Azure.AI.MetricsAd
     public partial class AzureBlobDataFeedSource : Azure.AI.MetricsAdvisor.Models.DataFeedSource
     {
         public AzureBlobDataFeedSource(string connectionString, string container, string blobTemplate) { }
+        public Azure.AI.MetricsAdvisor.Models.AzureBlobDataFeedSource.AuthenticationType? Authentication { get { throw null; } set { } }
```

Comment:
In general, the flow is:
- User creates a `DatasourceCredential` using a Create operation:
```cs
var credential = new ServicePrincipalDatasourceCredential("clientId", "clientSecret", "tenantId");
credential = adminClient.CreateDatasourceCredential(credential);
string credentialId = credential.Id;
```
- With the credential ID, they may create a data feed with a specific authentication for its data source:
```cs
var dataFeed = new DataFeed()
{
  DataSource = new SqlServerDataFeedSource("connString", "query")
  {
    Authentication = SqlServerDataFeedSource.AuthenticationType.ServicePrincipal,
    DatasourceCredentialId = credentialId
  }
}
dataFeed = adminClient.CreateDataFeed(dataFeed);
```

In particular, the Managed Identity authentication does not require a datasource credential.